src/main/resources/liquibase-changeLog-studentTable.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+  <changeSet id="st-1" author="Unknown">
+    <addColumn tableName="student">
+      <column name="incoming_test" type="INT"/>
+      <column name="entry_score" type="DOUBLE PRECISION"/>
+      <column name="approved_by_expert_id" type="INT" defaultValueNumeric="1">
+      <constraints nullable="false"/>
```

Comment:
as approved by field is not nullable, it must be specified when adding new student